src/main/java/com/google/step/servlets/JsonFormatter.java

```diff
@@ -102,7 +102,7 @@ public static String getDealListJson(List<Deal> deals) {
     Map<String, Object> restaurantMap = new HashMap<>();
     restaurantMap.put("id", restaurant.id);
     restaurantMap.put("name", restaurant.name);
-    restaurantMap.put("photoBlobkey", restaurant.photoBlobkey);
+    restaurantMap.put("image", getImageUrl(restaurant.photoBlobkey));
```

Comment:
since you changed this can help to change the restaurantservlettest and restaurantpostservlettest as well cos the json no longer matches